main/lsp/requests/completion.cc

```diff
@@ -63,10 +70,11 @@ const RubyKeyword rubyKeywords[] = {
     {"true", "Boolean true."},
     // This is also defined on Kernel
     // {"undef", "Prevents a class or module from responding to a method call."},
-    {"unless", "Used for unless and modifier unless expressions."},
-    {"until", "Creates a loop that executes until the condition is true."},
-    {"when", "A condition in a case expression."},
-    {"while", "Creates a loop that executes while the condition is true."},
+    {"unless", "Used for unless and modifier unless expressions.", "unless ${1:expr}\n  $0\nend"},
+    {"until", "Creates a loop that executes until the condition is true.", "until ${1:expr}\n  $0\nend"},
+    // Would really like to dedent the line too...
```

Comment:
Is that something that can be expressed with the snippets?

Reply:
I don't think so. This is probably something that'll have to wait for autoformat.